raqm.c

```diff
@@ -380,7 +384,7 @@ harfbuzz_shape (FriBidiChar* unicode_str,
     run->hb_buffer = hb_buffer_create ();
 
     /* adding text to current buffer */
-    hb_buffer_add_utf32 (run->hb_buffer, unicode_str, length, run->pos, run->len);
+    hb_buffer_add_utf32 (run->hb_buffer, unicode_str, length, (unsigned int)(run->pos), run->len);
```

Comment:
We can declare `pos` as an `nsigned int` to avoid the cast, it is only there for this function.